parquet/fuzz/fuzz_targets/parse_metadata.rs

```diff
@@ -0,0 +1,6 @@
+#![no_main]
```

Comment:
This target seems to be for parquet, and not arrow-rs.

Reply:
> Is this part of arrow-rs, or parquet?

this implementation of parsing the parquet format is part of this arrow-rs project. As I understand it, parquet is the file format, and arrow is the in-memory format.

You might be confused with [arrow2](https://github.com/jorgecarleitao/arrow2#faq)? which is a significantly divergent fork with a very different approach to the format.